app/main/routes.py

```diff
@@ -708,3 +722,31 @@ def terms_of_use():
 @bp.app_errorhandler(HTTPException)
 def http_exception(error):
     return render_template(f"{error.code}.html"), error.code
+
+
+@bp.route("/record/<uuid:record_id>/manifest")
+@access_token_sign_in_required
+def generate_manifest(record_id: uuid.UUID):
```

Comment:
would be really good to add 1 or 2 high level tests for this new endpoint and then any edge casey ones fdor manifest creation at the generate_pdf_manifest and generate_image_manifest function levels

Reply:
Added PDF generation & image generation manifest tests to ensure correct manifest is produced.